src/test/java/de/butbetter/keksys/CalculationControllerTests.java

```diff
@@ -17,4 +22,14 @@ void contextLoads() {
         assertThat(calcuationController).isNotNull();
     }
 
+    @Test
+    void sendParameter14and100() {
+        assertEquals(this.calcuationController.calculation(100, 14).getStatusCode(), HttpStatus.OK);
+    }
+
+    @Test
+    void sendParameterMinus1and100() {
+        assertEquals(this.calcuationController.calculation(-1, 100).getStatusCode(), HttpStatus.OK);
```

Comment:
Added already comment in Notefox PR.
I think negative values shouldnt be changed into 0, because the VAT of a price = 0 is also 0.
I think negative values should trigger a IllegalArgumentexception, which has to be caught.
If we make those changes this test would go wrong.

Reply:
This assert not tests the solution of calculation methode. This methode tested if REST request httpStatus.ok

Reply:
i know all im saying, once the changes are applied to PR of Notefox, this test should not return 200 but 400.

Reply:
@MathisHTW will be done on whe time comes
this is not part of this Pull-Request

Reply:
(Please add a Issue for that on Jira and we'll adress it)

Reply:
closed